File: src/question/month08/day0813/ms_0813.cpp
#include <iostream>

using namespace std;

int main() {
	int data[100], i = 0, count = 0;
	int max = -1000, min = 1000;

	while (i < 100) 
	{
		scanf_s("%d", &data[i]);

		if (-1000 < data[i] && data[i] < 1000) 
		{
			if (data[i] != 999)
				count++;
			else break;
		}
		else continue;
		
		i++;
	}

	for (i = 0; i < count; i++) 
	{
		if (max < data[i])
			max = data[i];

		if (min > data[i])
			min = data[i];
	}

	cout << "max : " << max << endl;
	cout << "min : " << min << endl;

	return 0;
}

File: year2020/month08/day0813/ms_0813.cs
using System;

namespace algorithm_34
{
    class ms_0813
    {
        static void Main(string[] args)
        {
            int[] data = new int[100];
            int i = 0, count = 0;

            while(i < data.Length) 
            {
                int digit = Convert.ToInt32(Console.ReadLine());

                if(-1000 < digit && digit < 1000) 
                {
                    if(digit != 999) 
                    {
                        data[i] = digit;
                        count++;
                    } 
                    else break;
                } 
                else continue;

                i++;
            }
            
            int max = -1000, min = 1000;

            for(i = 0; i < count; i++) 
            {
                if(max < data[i]) max = data[i];
                if(min > data[i]) min = data[i];
            }

            Console.WriteLine($"max : {max}\nmin : {min}");
        }
    }
}
